src/pages/Discussions/DiscussionListPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+import DiscussionsLayout from '../../components/composite/layout/Discussions/DiscussionsLayout';
+import { ThemeProviderV2 } from '../../context/ThemeProvider';
+import { useUpdateNavigation } from '../../hooks';
+import { DiscussionListView } from '../../views/Discussions/DiscussionsListView';
+import { PageProps } from '../common';
+
+interface DiscussionsPageProps extends PageProps {}
+
+export const DiscussionListPage: FC<DiscussionsPageProps> = ({ paths }) => {
+  useUpdateNavigation({ currentPaths: paths });
+
+  return (
+    <ThemeProviderV2>
+      <DiscussionsLayout title="Discussions" allowCreation>
```

Comment:
translation

Reply:
This should come from somewhere, still unknow.